config-provisioning/src/main/java/com/yahoo/config/provision/ClusterMembership.java

```diff
@@ -115,7 +115,7 @@ public static ClusterMembership from(String stringValue, Version vespaVersion) {
         return new ClusterMembership(stringValue, vespaVersion, Optional.empty());
     }
 
-    public static ClusterMembership from(String stringValue, Version vespaVersion, Optional<DockerImage> dockerImageRepo) {
```

Comment:
This also breaks that API since all of config-provisioning is an API for the config model, but based on current usage this one is probably ok.